src/src/Controller/DiskografieController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+use Doctrine\ORM\EntityManagerInterface;
+
+class DiskografieController extends AbstractController
+{
+    #[Route('/diskografie', name: 'app_diskografie')]
+    public function index(Request $request, EntityManagerInterface $entityManager): Response
```

Comment:
Neither `$request`, nor `$entityManager` are needed in this controller. You should remove the parameters.